### PR TITLE
Update tutorials with the recent changes

### PR DESCRIPTION
**Pull request by mindula**
## Purpose
$title.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: sending-emails-from-a-service/tests/test.bal [hunk 1]
```diff
@@ -3,7 +3,7 @@ import ballerina/http;
 import ballerina/random;
 import ballerina/test;
 
-final http:Client cl = check new (string `http://localhost:${port}/healthcare/categories`);
+final http:Client cl = check new (string `http://localhost:${8290}/healthcare/categories`);
```
Comment (MaryamZi) [inline sending-emails-from-a-service/tests/test.bal]:
```suggestion
final http:Client cl = check new ("http://localhost:8290/healthcare/categories");
```

File: sending-emails-from-a-service/tests/test.bal [hunk 2]
```diff
@@ -118,7 +118,7 @@ public client class MockHttpClient {
     isolated resource function post [http:PathParamType... path](http:RequestMessage message, 
             map<string|string[]>? headers = (), 
             string? mediaType = (), 
-            http:TargetType targetType = <>, *http:QueryParams params) 
+            http:TargetType targetType = http:Response, *http:QueryParams params)
```
Comment (MaryamZi) [inline sending-emails-from-a-service/tests/test.bal:121]: Did this compile previously?

File: service-orchestration/tests/tests.bal [hunk 3]
```diff
@@ -2,7 +2,7 @@ import ballerina/http;
 import ballerina/random;
 import ballerina/test;
 
-final http:Client cl = check new (string `http://localhost:${port}/healthcare/categories`);
+final http:Client cl = check new (string `http://localhost:${8290}/healthcare/categories`);
```
Comment (MaryamZi) [inline service-orchestration/tests/tests.bal]: Fix the URL to use a string literal.

File: sending-emails-from-a-service/README.md [hunk 4]
```diff
@@ -650,39 +582,15 @@ Let's send a request to the service using cURL as follows.
...
-        Name: thomas collins
-        Specialization: surgery
-
-    Payment Details
-        Doctor Fee: 7000.0
-        Discount: 20
-        Total Fee: 5600.0
-        Payment Status: settled
-```
+![Email](./resources/email.png)
```
Comment (MaryamZi) [inline sending-emails-from-a-service/README.md:593]: Broken.

**Review by dulajdilshan — DISMISSED** [review]

File: sending-emails-from-a-service/tests/test.bal (hunk 2 above)
Comment (mindula) [reply sending-emails-from-a-service/tests/test.bal:121]: Yes it did. After updating the version, got an error saying `cannot use an inferred typedesc default with a parameter on which the return type does not depend on`.

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**